cmd/gateway-azure.go

```diff
@@ -181,6 +156,65 @@ func azureToObjectError(err error, params ...string) error {
 	return e
 }
 
+func mustGetAzureUploadID() string {
+	var id [8]byte
+
+	n, err := io.ReadFull(rand.Reader, id[:])
+	fatalIf(err, "unable to generate upload ID for azure")
+	if n != len(id) {
+		fatalIf(fmt.Errorf("insufficient random data (expected: %d, read: %d)", len(id), n),
```

Comment:
fatalIf is a bad idea we should just panic() instead - http server will catch panics and gracefully handle it.

Reply:
Done